.github/install-clang-15.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
clang-15 is not printed in stone for our usage. Basically, once clang-16 will be released there will be big desire to switch to it. So, similar script will be needed. I suggest to generalize that and rename script to `install-clang.sh` and allow it accept the argument - version number. As such, we will be able to call:
```
./install-clang.sh 15
./install-clang.sh 16
```

Reply:
Existing code used hard-coded clang-15, so I followed suit. Making it more flexible sounds like a good idea though - hell one could even add few versions in the matrix.